scala/BUILD

```diff
@@ -0,0 +1,20 @@
+load("//scala:scala_toolchain.bzl", "scala_toolchain")
+
+#call scala toolchain
+scala_toolchain(
+      name = 'scala_toolchain_impl',
+      scalacopts = [],
+      visibility = ["//visibility:public"]
+    )
+    
+toolchain_type(
+    name = "scala_toolchain_type",
```

Comment:
Nit: Just "toolchain_type" is sufficient, since "scala" is already in the package name, and it'll make your toolchain type slightly smaller.

Ie,
@io_bazel_rules_scala//scala:scala_toolchain_type
vs
@io_bazel_rules_scala//scala:toolchain_type

This will also be consistent with the go, rust, cc, and jdk toolchain types.

Reply:
can we change to just `toolchain_type` as katre suggests?